resources/language/resource.language.en_gb/strings.po

```diff
@@ -506,6 +506,14 @@ msgctxt "#30440"
 msgid "A-Z"
 msgstr ""
 
+msgctxt "#30450"
+msgid "Update add-on repositories"
+msgstr ""
+
+msgctxt "#30451"
+msgid "If a VRT NU add-on update is available, it will now be updated and you will be notified.\nMake sure the VRT NU add-on has [I]Auto-update[/I] enabled."
```

Comment:
What do you mean with "Make sure the VRT NU add-on has Auto-update enabled."?
Isn't this a global Kodi setting?

Reply:
No, every add-on has a switch to enable/disable auto-update. We have seen that if you install an add-on from a ZIP file (that wasn't updated) this switch is off. But in most cases it will be on.

The problem is that in the OK window I only have 4 lines, so I cannot add a lot of information otherwise not everything is visible. Quite annoying.

Reply:
I have disabled the global auto-update setting and then this individual add-on switch is not displayed. I think it's better to change the text to:

"Make sure you have _Auto-update_ enabled in Kodi and the VRT NU add-on."

Reply:
Good suggestion! Having an **Updates** category for just one item is not very good, but burying it among other unrelated options in **Expert** is worse IMO. So I will do it like this for now, we can improve on this later.

Reply:
BTW we can detect if the global setting is disabled, and we could offer to enable it (although I am not sure if this would be accepted by Team Kodi. We need to check. At least we can inform the user if this is a problem and maybe open the settings.

This is a future improvement.